src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  position: relative;
+}
+
+.search__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  padding-left: 62px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 800;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  background: linear-gradient(180deg, #fff 0, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+}
+
+.search__button {
+  position: absolute;
+  left: 26px;
+  border: none;
+  background-color: transparent;
+  top: 25px;
+  font-family: "Avenir", sans-serif;
+  padding: 0;
+  cursor: pointer;
+}
+
+.search__button img {
+  width: 19px;
+  height: 19px;
+}
+
+.search__input-small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search__button-small {
+  left: 13px;
+  top: 15px;
+}
+
+.search__button-small img {
```

Comment:
add class for image, do not increase specific and use tag selector just for reset margin and padding